lib/sqlbuilder/insert.go

```diff
@@ -230,7 +244,10 @@ func (ins *inserter) build() (*inserterQuery, error) {
 		return nil, err
 	}
 	ret := iq.(*inserterQuery)
-	ret.values, ret.arguments = ret.processValues()
+	ret.values, ret.arguments, err = ret.processValues()
```

Comment:
since `processValues` is called on `ret` wouldn't it make more sense for it to modify `ret` internally and return error only?